source/gameengine/GamePlayer/GPG_ghost.cpp

```diff
@@ -938,8 +938,30 @@ int main(int argc,
 #endif
   UI_theme_init_default();
 
-  UserDef *user_def = BKE_blendfile_userdef_from_defaults();
-  BKE_blender_userdef_data_set_and_free(user_def);
+  /* Try to load existing user preferences from config folder:
+   * Either from exported Game folder, either from blender userprefs file
+   */
+  char filepath_startup[FILE_MAX] = "";
+  char filepath_userdef[FILE_MAX] = "";
+  UserDef *userdef = nullptr;
+
+  const char *const cfgdir = BKE_appdir_folder_id(BLENDER_USER_CONFIG, NULL);
+  if (cfgdir) {
+    BLI_path_join(filepath_startup, sizeof(filepath_startup), cfgdir, BLENDER_STARTUP_FILE, NULL);
+    BLI_path_join(filepath_userdef, sizeof(filepath_startup), cfgdir, BLENDER_USERPREF_FILE, NULL);
```

Comment:
filepath_startup is really used?

Additionally, I think there is typo as it should be:
BLI_path_join(filepath_userdef, sizeof(**filepath_userdef**), cfgdir, BLENDER_USERPREF_FILE, NULL);